tests/test_flake8.py

```diff
@@ -0,0 +1,33 @@
+from unittest import TestCase
```

Comment:
Rather than remove this test, could you move it into `integration_tests/`, then add the line

```
  pytest integration_tests/test_flake8.py
```

To the `tox.ini`, as part of the `pre-commit` command?  This is where integration tests live, and flake8 is already installed there for compatibility testing.  You can then test that it passes by running

```
tox -e pre-commit
```

(Although, you may want to comment out the other integration tests when testing it locally, since you'd need the fixtures present to run them.)